database/rrdcalctemplate.c

```diff
@@ -4,6 +4,50 @@
 #include "rrd.h"
 
 // ----------------------------------------------------------------------------
+
+static int rrdcalctemplate_has_label(RRDCALCTEMPLATE *rt,  RRDHOST *host) {
```

Comment:
This is a misleading name for the function.

Reply:
Done!